configs/compy/config.20190711.testLuke.A_WCYCL1850S.ne30_oECv3.compy

```diff
@@ -0,0 +1,138 @@
+[runs]
+## options related to the run to be analyzed and control runs to be
+## compared against
+
+# mainRunName is a name that identifies the simulation being analyzed.
+mainRunName = 20190711.testLuke.A_WCYCL1850S.ne30_oECv3.compy 
+
+# config file for a control run to which this run will be compared.  The
+# analysis should have already been run to completion once with this config
+# file, so that the relevant MPAS climatologies already exist and have been
+# remapped to the comparison grid.  Leave this option commented out if no
+# control run is desired.
+# controlRunConfigFile = /path/to/config/file
+
+# config file for a main run on which the analysis was already run to
+# completion.  The relevant MPAS climatologies already exist and have been
+# remapped to the comparison grid and time series have been extracted.
+# Leave this option commented out if the analysis for the main run should be
+# performed.
+mainRunConfigFile = config.GM_20190605_GMtest_Kappa3d_vary_c2_intel.compy 
+
+[execute]
+## options related to executing parallel tasks
+
+# the number of parallel tasks (1 means tasks run in serial, the default)
+parallelTaskCount = 6
+
+# the parallelism mode in ncclimo ("serial" or "bck")
+# Set this to "bck" (background parallelism) if running on a machine that can
+# handle 12 simultaneous processes, one for each monthly climatology.
+ncclimoParallelMode = bck
+
+[diagnostics]
+## config options related to observations, mapping files and region files used
+## by MPAS-Analysis in diagnostics computations.
+
+# The base path to the diagnostics directory.  Typically, this will be a shared
+# directory on each E3SM supported machine (see the example config files for
+# its location).  For other machines, this would be the directory pointed to
+# when running "download_analysis_data.py" to get the public observations,
+# mapping files and region files.
+baseDirectory = /compyfs/diagnostics
+
+[input]
+## options related to reading in the results to be analyzed
+
+# directory containing model results
+baseDirectory = /compyfs/wolf966/e3sm_scratch/20190711.testLuke.A_WCYCL1850S.ne30_oECv3.compy/run 
+
+# names of ocean and sea ice meshes (e.g. oEC60to30v3, oQU240v3, oRRS30to10v3, etc.)
+mpasMeshName = oEC60to30v3
+
+# names of namelist and streams files, either a path relative to baseDirectory
+# or an absolute path.
+#oceanNamelistFileName = mpas-o_in
+#oceanStreamsFileName = streams.ocean
+#seaIceNamelistFileName = mpas-cice_in
+#seaIceStreamsFileName = streams.cice
+
+[output]
+## options related to writing out plots, intermediate cached data sets, logs,
+## etc.
+
+# directory where analysis should be written
+baseDirectory = /compyfs/vanr302/case_output/20190711.testLuke.A_WCYCL1850S.ne30_oECv3.compy_yr_30_40
```

Comment:
```suggestion
baseDirectory = /path/to/output/directory
```
We usually put nonsense here so people don't accidentally write output to your directory.

Reply:
Thanks, great point.  This is changed.